fix-formatting.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
Should we apply the same fixes here? E.g have only fix-formatting, not run `buildifier` separately.